userprofile/index.html

```diff
@@ -96,12 +132,63 @@ <h4 class="list-group-item-heading"><small>{{userData.screen_name}}</small></h4>
     </center>
 
     <script>
+=======
+<center>
+<h1>Enter the username of the user</h1>
+<div ng-app="TweetSearch" ng-controller="Controller">
+<input type="text" id="query" ng-model="query" placeholder="Search tweet from Loklak">
+<a href="#" id="button" ng-click="Search()">Search</a>
+<br>
+<br>
+<br>
+<div class="container">
+<div class="row">
+<div class="col-md-4">
+<div id="contactcard">
+<div class="panel panel-default">
+<div class="panel-heading clearfix">
+<h3 class="panel-title pull-left">User Profile</h3>
+</div>
+<div class="list-group">
+<div class="list-group-item">
+<img src="{{userData.profile_image_url}}" style="pull-left">
+<h4 class="list-group-item-heading">{{userData.name}}</h4>
+</div>
+<div class="list-group-item">
+<h4 class="list-group-item-heading"><strong>{{userData.followers_count}}</strong> Followers</h4>
+</div>
+<div class="list-group-item">
+<h4 class="list-group-item-heading"><strong>{{userData.statuses_count}}</strong> Statuses</h4>
+</div>
+<div class="list-group-item">
+<h4 class="list-group-item-heading"><small>{{userData.screen_name}}</small></h4>
+</div>
+</div>       
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/userprofile/index.html
+++ b/userprofile/index.html
@@ -163,7 +163,7 @@
 <div class="list-group-item">
 <h4 class="list-group-item-heading"><small>{{userData.screen_name}}</small></h4>
 </div>
-</div>       
+</div>
 </div>
 </div>
 <!-- <p>{{userData}}</p> -->
```